xml/ha_config_basics.xml

```diff
@@ -657,9 +651,11 @@
       </listitem>
      </varlistentry>
      <varlistentry>
-      <term>Stateful Clones (Multi-state Resources)</term>
+      <term>Promotable Clones (Multi-state Resources)</term>
       <listitem>
        <para>
+        <remark>taroth 2018-09-03: ygao, is the following still correct? if not,
+        how to adjust the following text? please advise!</remark>
         Active instances of these resources are divided into two states,
         active and passive. These are also sometimes called primary and
         secondary, or master and slave. Stateful clones can be either
```

Comment:
Promotable clones

Reply:
Will switch to the term 'promotable clones' (instead of 'multi-state resources') throughout the docs consistently (and explain at first appearance that  they have been called 'multi-state resources' before). @gao-yan: How about the states? Can we still call them 'master' and 'slave'?? (that's the terminology we used throughout the docs so far)